### ISSUE TITLE
Astropy 4.0.1 support

### ISSUE THREAD

**Issue by nabobalis** [astropy/reproject#224]
If I have version 4.0.0 or <4.0.0 of astropy installed, I can import reproject but with the first bug release I have an import error:

```python
>>> from reproject import reproject_interp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/miniconda/envs/sunpy/lib/python3.8/site-packages/reproject/__init__.py", line 13, in <module>
    from .interpolation import reproject_interp  # noqa
  File "/opt/miniconda/envs/sunpy/lib/python3.8/site-packages/reproject/interpolation/__init__.py", line 5, in <module>
    from .high_level import *  # noqa
  File "/opt/miniconda/envs/sunpy/lib/python3.8/site-packages/reproject/interpolation/high_level.py", line 18, in <module>
    def reproject_interp(input_data, output_projection, shape_out=None, hdu_in=0,
  File "/opt/miniconda/envs/sunpy/lib/python3.8/site-packages/astropy/utils/decorators.py", line 443, in decorator
    param = arguments[old_name[i]]
KeyError: 'independent_celestial_slices'
```

**Comment by pllim** [comment]
Please see discussions at astropy/astropy#10084

cc @bsipocz

**Comment by astrofrog** [comment]
Fixed in reproject v0.7: https://pypi.org/project/reproject/0.7/

**Comment by nabobalis** [comment]
Thanks!
